src/_layouts/page.njk

```diff
@@ -20,8 +25,14 @@ layout: base.njk
         </div>
     </div>
     <div class="lg:w-[30%] w-[80%] lg:mt-60 flex items-center justify-center mx-auto">
-        <img src="{{ env.baseUrl }}img/assets/laptop.svg" alt="Guide" class="w-[40%] lg:w-[60%] transition-transform duration-100" style="animation: floating 2s ease-in-out infinite;">
-        <img src="{{ env.baseUrl }}img/assets/filter-coffee.svg" alt="Guide" class="w-[40%] lg:w-[60%] transition-transform duration-100" style="animation: floating 2s ease-in-out infinite;">
+        {% for icon in iconset %}
+            <img 
```

Comment:
What if someone mistakenly gives more 2 icons which don't go well with the layout?

Reply:
prepared pre-defined iconset consisting of 1 or 2 icons.